src/views/homepage/components/merchandising-slots/slots/hcp-slot/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { LogoCtaSlot } from '../logo-cta-slot'
+import logo from '@hashicorp/mktg-logos/product/hcp/primary/black.svg?include'
+import { HcpSlotProps } from './types'
+import s from './hcp-slot.module.css'
+
+function HcpSlot({ url, cardTitle, description, ctaText }: HcpSlotProps) {
+	return (
+		<LogoCtaSlot
```

Comment:
I've set up a `LogoCtaSlot`. It handles the logo positioning and most other display concerns for both the HCP and Vault slots. The specific slots can customize background elements, logo SVG src, and description max-width.

Probably not the perfect abstraction, but felt like a good balance to meet our short-term needs here.